projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html

```diff
@@ -52,7 +52,7 @@
                 <span
                     #ghost
                     class="t-ghost"
-                    [innerText]="formattedCard.slice(-tailLength)"
+                    [innerHTML]="formattedCard: tuiSlice: -tailLength"
```

Comment:
jest-angular-preset doesn't support `innerText` in compile time
https://github.com/Tinkoff/taiga-ui/issues/3800

<img width="1090" alt="image" src="https://user-images.githubusercontent.com/12021443/232549178-07ad5c50-d018-4171-9d10-5b8c3af750fd.png">